diff --git a/node_modules/react-native-tab-view/src/PagerViewAdapter.tsx b/node_modules/react-native-tab-view/src/PagerViewAdapter.tsx
index 4dca513..70c3453 100644
--- a/node_modules/react-native-tab-view/src/PagerViewAdapter.tsx
+++ b/node_modules/react-native-tab-view/src/PagerViewAdapter.tsx
@@ -111,7 +111,7 @@ export default function PagerViewAdapter<T extends Route>({
 
           offset.removeListener(subscription);
         });
-
+        // offset.removeListener(subscription); //add this line
         onSwipeStart?.();
         return;
       }
